exporter/metric/exporter.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 1999-2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+import (
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/alibaba/sentinel-golang/core/config"
+	"github.com/alibaba/sentinel-golang/exporter/metric/prometheus"
+)
+
+var (
+	exporter Exporter
+
+	host      string
+	app       string
+	pid       string
+	namespace string
+)
+
+func init() {
+	exporter = newPrometheusExporter()
```

Comment:
We might need to provide func to reset Exporter

Reply:
May be supported in future.